### ISSUE TITLE
Stop copying `moment_kinetics_input.jl`?

### ISSUE THREAD

**Issue by mrhardman** [mabarnes/moment_kinetics#294]
Now that the inputs are all (?) passed through checked namelists using `set_defaults_and_check_section!()`, would it be possible to stop copying the `moment_kinetics_input.jl` file to the run directory? `moment_kinetics_input.jl` should now be as static as any other part of the source.

**Comment by johnomotani** [comment]
I agree there's no need to copy `moment_kinetics_input.jl` now. Would be more useful to copy the TOML input file (although its contents should be captured in the .h5 output files).

**Comment by mrhardman** [comment]
It looks like we can just turn off the existing copy by modifying the appropriate places `save_inputs_to_txt=true` to `save_inputs_to_txt=false`. The copying of the `.toml` is a little more challenging because we don't propagate the file path into the function where the inputs are read. @johnomotani Do you have a preferred solution for this? This is the kind of problem where arbitrary choices get made.

**Comment by johnomotani** [comment]
`save_inputs_to_txt` is also the flag that is used to write the `moment_kinetics.input` file. I'm not sure how useful that one is either - it could probably do with being cleaned up a bit and made more logical - but I wouldn't suggest modifying it at the moment.
Why not just delete https://github.com/mabarnes/moment_kinetics/blob/eac094f4346020edcc445e8e1f883edb79b1ba00/moment_kinetics/src/moment_kinetics_input.jl#L429-L432
and remove the `save_inputs_to_txt` argument from `check_input()`?

Agreed I don't see a nice way of copying the TOML. I'd probably add the path to the TOML file to the `input_dict`, and then pop it out once the file is copied.
